test/integration/lib/config/foxbox.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+
+	nupnp_server : {
```

Comment:
strange, I changed the tab setting to 2 spaces, but here it still shows up as the same.